tests/upgrade/basic/task.yaml

```
@@ -12,6 +12,17 @@ restore: |
     rm -f /var/tmp/myevil.txt
     rm -f ./do-classic ./prevsnapdver.out
 
+    # an older version of snapd might have been missing proper context
```

Please capitalise sentences.

```suggestion
    # An older version of snapd might have been missing proper context
```